We need QtCore to satisfy symbols, the ebuild sets CC=CXX, because we
need g++ to properly compile C++ code as well.

--- configure
+++ configure
@@ -128,6 +128,7 @@
 subdirs="/qt/Qt /qt4/Qt /qt4"
 search_includes qobject.h || err "The QT Library headerfiles were not found. Set QTDIR appropriately."
 search_lib QtGui4 QtGui || err "The QT library was not found."
+search_lib QtCore4 QtCore || err "The QT library was not found."
 search_lib c_r || true
 
 ###################### OpenSSL
@@ -218,7 +219,7 @@
 EOF
 
 find_make() {
-  for dirs in /usr/local/bin /sw/bin /bin /usr/bin; do
+  for dirs in "$prefix"/usr/bin; do
     for make in gmake make; do
       if "${dirs}/${make}" -v 2>/dev/null | grep GNU; then
 	mak="${dirs}/${make}"
